mypy/test/testtypes.py

```diff
@@ -451,6 +459,13 @@ def test_false_only_of_union(self) -> None:
             assert fo.items[0].can_be_false
             assert fo.items[1] is tup_type
 
+    def test_false_only_of_truthy_type_is_uninhabited(self) -> None:
```

Comment:
I think that it will be more useful to add tests to some `check-*.test` file.

I think that these ones can be removed.